docs/source/platform/operation-registry.md

```diff
@@ -118,7 +118,7 @@ Currently, once an operation is registered it will remain registered indefinitel
 
 If you encounter any errors, check the _**Troubleshooting**_ section below.
 
-####3.1 Optionally, set the schema tag  
```

Comment:
It might be nice to elucidate the **why** or link to another article, rather than only the **how**